File: SUMA/Suma.py
class Suma(object):

    def __init__(self, a, b):
        self.__a = a
        self.__b = b
        self.set_a(a)
        self.set_b(b)
    def set_a(self, a):
        self.__a = a
    def set_b(self, b):
        self.__b = b
    def get_a(self):
        return self.__a
    def get_b(self):
        return self.__b
    def Sumar(self):
        return self.__a + self.__b

File: SUMA/Suma.java
/**
 * Suma
 */
 public class Suma { 
    private double a, b; 
    public Suma(double a, double b) {
        this.a = a; 
        this.b = b; 
        set_a(a); 
        set_b(b); 
    }
 
    public void set_a(double a) { 
        this.a = a; 
    } 
    public void set_b(double b) {
        this.b = b; 
    } 
    public double get_a() { 
        return a; 
    } 
    public double get_b(){
        return b; 
    }
 
    public double Sumar() { 
        return a + b; 
    }
 
}